anms-ui/public/app/components/management/agents/reports.vue

```diff
@@ -52,7 +53,7 @@ export default {
       this.loading = true;
       this.tableHeaders = [];
       this.tableItems = [];
-      if (this.reports[this.selected] == undefined) {
+      // if (this.reports[this.selected] == undefined) {
```

Comment:
Commented code is wanted here and below?

Reply:
yeah it was unwanted  just removed it